Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -935,20 +927,15 @@ class CombinatorialKalmanFilter {
     /// @param prevTip The index of the previous state
     ///
     /// @return The tip of added state
-    std::size_t addNonSourcelinkState(const TrackStatePropMask& stateMask,
+    std::size_t addNonSourcelinkState(TrackStatePropMask stateMask,
```

Comment:
another API change that might count as a breaking change, but again probably won't actually break any real code.

Reply:
I think this should be unproblematic. The mask is actually an 8 bit value so it does not make sense to reference to it.